components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/OpenIDConnectClaimFilterImpl.java

```diff
@@ -231,6 +220,64 @@ public Map<String, Object> getClaimsFilteredByUserConsent(Map<String, Object> us
         return userClaims;
     }
 
+    /**
+     * Handles filtering claims of the address scope based on user consent as they are added as
+     * sub-claims of address claim.
```

Comment:
Shall we simplify the method comment in a more readable way and point the spec also?

Eg:

Handles filtering the fields of address claim from the user consented claims https://openid.net/specs/openid-connect-core-1_0.html#AddressClaim